WebXR API.md

```diff
@@ -79,6 +81,10 @@ A Hololens could expose a single passthrough display.
 		long requestFrame(XRFrameRequestCallback callback);
 		void cancelFrame(long handle);
 
+		// Request the camera frame available in XRPresentationFrame.
+		// This operation will ask for users' permission.
+		Promise<void> requestCameraAccess(XRCameraSource camera);
```

Comment:
Request camera frame will ask for users' permission. Without permission, the camera frame will not be available in presentation frame.